File: mario_more.py
def get_height():
    while True:
        try:
            height = int(input("Height: "))
            if 1 <= height <= 8:
                return height
        except ValueError:
            pass


def main():
    height = get_height()
    for row in range(1, height + 1):
        pyramid = " " * (height - row) + "#" * row
        print(f"{pyramid}  {pyramid}")


if __name__ == "__main__":
    main()

File: mario_more.c
#include <stdio.h>

void print_pyramid(int height, int row);

int main(void)
{
    int height;

    do
    {
        printf("Height: ");
        scanf("%d", &height);
    }
    while (height < 1 || height > 8);

    for (int row = 1; row <= height; row++)
    {
        print_pyramid(height, row);
        printf("  ");
        print_pyramid(height, row);
        printf("\n");
    }
}

void print_pyramid(int height, int row)
{
    for (int col = 0; col < height - row; col++)
    {
        printf(" ");
    }
    for (int col = 0; col < row; col++)
    {
        printf("#");
    }
}
